udata_front/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr.po

```diff
@@ -1516,15 +1524,19 @@ msgstr "Cet article est un brouillon et ne sera pas visible par les autres utili
 msgid "This post has been deleted. This will be permanent in the next 24 hours"
 msgstr "Cet article a été supprimé. Cel sera permanent dans les prochaines 24h."
 
-#: udata_front/theme/gouvfr/templates/post/display.html:111
+#: udata_front/theme/gouvfr/templates/post/display.html:106
+msgid "Navigate in the post collection"
+msgstr "Naviguer dans la collection d'actualités"
```

Comment:
Not sure about this translation!

Reply:
Do we want to remove the "collection" term ? Like "Navigate to other posts"

Reply:
The current wording and translations are from the audit report.

Reply:
I've updated the translation only to `Naviguer parmi les actualités`